math/flint/Portfile

```diff
@@ -4,11 +4,11 @@ PortSystem              1.0
 PortGroup               muniversal 1.0
 
 name                    flint
-version                 2.5.2
-revision                1
+version                 2.7.1
+revision                0
 categories              math devel
 # N.B. the next version will be LGPL-2.1+
```

Comment:
This comment can I guess now be removed

Reply:
Thank you for your comment.
I've removed the comment. Please have a look.